pkg/pipeline/controller.go

```diff
@@ -264,7 +264,7 @@ func (c *Controller) UpdateStream(ctx context.Context, req *livekit.UpdateStream
 	// add stream outputs first
 	for _, rawUrl := range req.AddOutputUrls {
 		// validate and redact url
-		url, redacted, err := c.ValidateUrl(rawUrl, types.OutputTypeRTMP)
+		url, redacted, err := config.ValidateUrl(rawUrl, types.OutputTypeRTMP)
```

Comment:
I have a question about this one. It's called for each output URL and it may eventually call `updateTwitchURL`. How likely is that its called twice? I guess it's not typical to output to multiple Twitch streams, right?

Reply:
I've never seen anyone do it. I considered caching the results but it's a pretty cheap call and didn't seem worth the added complexity